app/src/main/java/com/scouts/kitchenplaner/model/usecases/CreateShoppingList.kt

```diff
@@ -54,6 +55,12 @@ class CreateShoppingList @Inject constructor(
      * Creates a Shopping List Entry with the specified name, amount and unit. Note that the created
      * entry is not yet persisted in the data base (create a shopping list via [createShoppingList]
      * to do so)
+     *
+     * @param name The name of the needed thing
```

Comment:
can we specify "thing" a bit more? I would generally hope that a variable ```name``` represents the name of some "thing" even without the doc comment...

Reply:
I'm not sure how, because the name is literally the description or name of the thing to buy. This can be an ingredient or anything else...

Reply:
"The name of the item that should be bought"?